src/main/java/org/reaktivity/nukleus/amqp/internal/stream/AmqpServerFactory.java

```diff
@@ -2388,31 +2534,29 @@ private void doNetworkReset(
 
             cleanupDecodeSlotIfNecessary();
 
-            doReset(network, routeId, initialId, traceId, authorization, EMPTY_OCTETS);
+            doReset(network, routeId, initialId, initialSeq, initialAck, bufferPool.slotCapacity(), traceId, authorization,
+                EMPTY_OCTETS);
         }
 
         private void doNetworkWindow(
             long traceId,
             long authorization,
-            int credit,
             int padding,
+            int maximum,
             long budgetId)
         {
-            assert credit > 0;
-
             state = AmqpState.openInitial(state);
 
-            initialBudget += credit;
-            doWindow(network, routeId, initialId, traceId, authorization, budgetId, credit, padding, 0);
+            doWindow(network, routeId, initialId,  initialSeq, initialAck, maximum, traceId, authorization, budgetId, padding, 0);
```

Comment:
Please see https://github.com/reaktivity/nukleus-kafka.java/blob/feature/flow-control-ack/src/main/java/org/reaktivity/nukleus/kafka/internal/stream/KafkaClientProduceFactory.java#L1035 for an example of modeling `doXXXWindow` for initial stream.